mvg/mvg.py

```diff
@@ -471,6 +515,75 @@ def create_measurement(
             "post", f"/sources/{sid}/measurements", do_not_raise, json=meas_struct
         )
 
+    def create_spectrum_measurement(
+        self,
+        sid: str,
+        freq_range: FrequencyRange,
+        timestamp: int,
+        data: Dict[str, List[float]],
+        meta: dict = None,
+        exist_ok: bool = False,
+    ):
+        """Stores a measurement on the server side.
+
+        Although it is up to the client side to handle the
+        scaling of data it is recommended that the values
+        represent the acceleration in g.
+        The timestamp shall represent the time when the measurement was
+        recorded.
```

Comment:
@tuix Could you review this description? This is a copy of the description for `create_measurement`. For spectrum measurement, do we need to provide this recommendation that the spectrum data could be in units of 
acceleration?